syntax = "proto3";

message UpdateGameStateRequestModel {
	int32 sessionId = 1;
}
